lib/pbench/test/unit/server/test_utils.py

```diff
@@ -35,3 +38,23 @@ def test_filesize_bytes():
             ), f"string '{size}', improperly converted to {res}, expected {exp_val}"
         with pytest.raises(Exception):
             res = filesize_bytes("bad")
+
+
+class TestUtcTimeHelper:
+    @freeze_time("1970-01-01")
+    def test_naive(self):
+        d = UtcTimeHelper(datetime.datetime.now())
+        assert d.to_iso_string() == "1970-01-01T00:00:00+00:00"
```

Comment:
Rather than relying on `freeze_time`, wouldn't it be better to just use the standard library?
```
    def test_naive(self):
        the_date = "1970-01-01T00:00:00"
        d = UtcTimeHelper(datetime.datetime.fromisoformat(the_date))
        assert d.to_iso_string() == the_date + "+00:00"

    def test_aware_utc(self):
        the_date = "2002-05-16T10:23:00+00:00"
        d = UtcTimeHelper(datetime.datetime.fromisoformat(the_date))
        assert d.to_iso_string() == the_date
```
In fact, it looks like this could be parameterized by providing parameters for input and output strings.

Reply:
This felt more natural at the time; arguably it doesn't really matter since the class isn't creating the timestamp.

And, FYI, `datetime.datetime.fromisoformat` is new in Python 3.7; we're still supporting Python 3.6 (default on RHEL 8.4), so this required `dateutil.parser`. Ah well...